crates/chain/tests/test_tx_graph_conflicts.rs

```diff
@@ -319,6 +340,41 @@ fn test_tx_conflict_handling() {
                 confirmed: Amount::from_sat(50000),
             },
         },
+        Scenario {
+            name: "confirmed tx conflits with unconfirmed",
```

Comment:
There's a little documentation typo:

```suggestion
            name: "confirmed tx conflicts with unconfirmed",
```

Reply:
Fixed.